demos/theme/index.html

```diff
@@ -466,124 +722,190 @@ <h2 class="mdc-typography--headline demo-component-section__heading">
-         * Drawer
-         */
 
-        var drawerEl = document.querySelector('.mdc-temporary-drawer');
-        var drawer = new mdc.drawer.MDCTemporaryDrawer(drawerEl);
+          /*
+           * Drawer
+           */
 
-        document.querySelector('.demo-menu-action').addEventListener('click', function() {
-          drawer.open = true;
-        });
+          var drawerEl = document.querySelector('.mdc-temporary-drawer');
+          var drawer = new mdc.drawer.MDCTemporaryDrawer(drawerEl);
 
-        drawerEl.addEventListener('MDCTemporaryDrawer:open', function() {
-          console.log('Received MDCTemporaryDrawer:open');
-        });
+          document.querySelector('.demo-menu-action').addEventListener('click', function() {
+            drawer.open = true;
+          });
 
-        drawerEl.addEventListener('MDCTemporaryDrawer:close', function() {
-          console.log('Received MDCTemporaryDrawer:close');
-        });
+          drawerEl.addEventListener('MDCTemporaryDrawer:open', function() {
+            console.log('Received MDCTemporaryDrawer:open');
+          });
 
-        /*
-         * Icon Toggle
-         */
+          drawerEl.addEventListener('MDCTemporaryDrawer:close', function() {
+            console.log('Received MDCTemporaryDrawer:close');
+          });
 
-        var iconToggleEls = getAll('.mdc-icon-toggle');
-        for (var i = 0, iconToggleEl; iconToggleEl = iconToggleEls[i]; i++) {
-          mdc.iconToggle.MDCIconToggle.attachTo(iconToggleEl);
-        }
+          /*
+           * Icon Toggle
+           */
 
-        /*
-         * Linear Progress
-         */
+          var iconToggleEls = getAll('.mdc-icon-toggle');
+          for (var i = 0, iconToggleEl; iconToggleEl = iconToggleEls[i]; i++) {
+            mdc.iconToggle.MDCIconToggle.attachTo(iconToggleEl);
+          }
 
-        var progressEls = getAll('.mdc-linear-progress');
-        for (var i = 0, progressEl; progressEl = progressEls[i]; i++) {
-          var linearProgress = mdc.linearProgress.MDCLinearProgress.attachTo(progressEl);
-          linearProgress.progress = 0.5;
-          if (progressEl.dataset.buffer) {
-            linearProgress.buffer = 0.75;
+          /*
+           * Linear Progress
+           */
+
+          var progressEls = getAll('.mdc-linear-progress');
+          for (var i = 0, progressEl; progressEl = progressEls[i]; i++) {
+            var linearProgress = mdc.linearProgress.MDCLinearProgress.attachTo(progressEl);
+            linearProgress.progress = 0.5;
+            if (progressEl.dataset.buffer) {
+              linearProgress.buffer = 0.75;
+            }
           }
+
+          /*
+           * Ripple
+           */
+
+          getAll('.mdc-ripple-surface:not([data-demo-no-js])').forEach(function(surface) {
+            mdc.ripple.MDCRipple.attachTo(surface);
+          });
+
+          /*
+           * Select
+           */
+
+          getAll('.mdc-select:not(select)').forEach(function(select) {
```

Comment:
Is :not(select) actually filtering anything?

Reply:
No, but I figured it wouldn't hurt to future-proof the selector in case we add a CSS-only example in the future.